aten/src/ATen/native/cuda/Loops.cuh

```diff
@@ -310,4 +333,4 @@ void gpu_kernel_multiple_outputs(TensorIteratorBase& iter, const func_t& f) {
   gpu_kernel_multiple_outputs_impl(iter, f);
 }
 
-}} //namespace at::native
+} //namespace at::native
```

Comment:
was this intentional?

Reply:
No, I like nested namespaces, it's a neat C++17 feature :)